release/ray_release/test.py

```diff
@@ -56,6 +57,13 @@ def get_byod_image_tag(self) -> str:
             os.environ["BUILDKITE_COMMIT"],
         )
         ray_version = commit[:6]
+        branch = os.environ.get("BUILDKITE_BRANCH", "")
+        assert branch == "master" or branch.startswith(
+            "releases"
+        ), f"Invalid branch name {branch}"
+        if branch.startswith("releases"):
+            release_name = re.search("[0-9]+\.[0-9]+\.[0-9].*", branch).group(0)
+            ray_version = f"{release_name}.{ray_version}"
```

Comment:
logic to get tag name is obtained from https://github.com/ray-project/ray/blob/master/ci/build/build-docker-images.py#L609

Reply:
I am killing that then: https://github.com/ray-project/ray/pull/35935/files

no one can stand in my way.. muahahahaha~~